ionic2/app/components/poke-poi-card/poke-poi-card.component.html

```diff
@@ -0,0 +1,43 @@
+<div class="slide-card" #slideCard @slide="slideState">
+  <div class="poke-poi-card-content" *ngIf="pokePOI" [ngClass]="'poi-type-'+pokePOI.getType()">
+    <ion-row>
+
+      <ion-col class="poke-column">
+        <ion-row>
+          <poke-poi-bubble [pokePOI]="pokePOI"></poke-poi-bubble>
+        </ion-row>
+        <ion-row>
+          <poke-type [type]="pokePOI.pokemon.types"></poke-type>
+        </ion-row>
+      </ion-col>
+
+      <ion-col style="flex-grow: 1;">
+        <ion-row>
+          <span class="poke-name">{{ pokePOI.pokemon.name }}</span>
+          <span class="poke-id">#{{ pokePOI.pokemon.pokemonId }}</span>
+        </ion-row>
+        <ion-row class="poke-description">
+          {{ pokePOI.pokemon.description }}
+        </ion-row>
+
+
+        <ion-row class="action-button-bar">
+          <button clear (click)="launchDirections()">
+            <ion-icon name="navigate"></ion-icon>
+            Directions
+          </button>
+
+          <button clear (click)="launchPokeDex()">
+            <ion-icon name="book"></ion-icon>
+            PokeDex
+          </button>
+        </ion-row>
+      </ion-col>
+    </ion-row>
+
+    <div class="poke-poi-type-line"></div>
```

Comment:
I think the type-line is nice when we show predictions and mobs in addition to sightings because then the color of the line changes as well.  I think it's just a nice visual feature.
Regarding rounded-corner badge, there's something about the rectangular version of yours that looks weird to me. It just looks like it doesn't belong to the card this way. I don't know.. it's difficult to argue about design decisions :D